changelogs/fragments/77247-get_url-doc-emphasis.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
```suggestion
trivial:
```

Reply:
What is this `trivial` category? It is not listed in https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment so I did not know it even exists!

Reply:
Huh.. did not realize it wasn't documented. Some collection owners use it when they require a changelog fragment for every PR. It's here - https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs.  `trivial` fragments won't appear in the changelog file.  I attempted that change until someone told me I could delete the file here in the PR.
I'll have to ask and see if it works for core changelogs as well...